src/notify.js

```diff
@@ -59,6 +63,14 @@ function Notify(title, options = {}) {
     }
 }
 
+Notify.serviceWorkerSupported = function() {
```

Comment:
To further emphasize, it's not even obvious that you want to notify your users when using the Push API. I guess it'd be rather common, but I could come up with use cases where Push is used and Notifications aren't...